design/object-bucket/object-bucket.md

```diff
@@ -45,6 +43,7 @@ The admin ...
 
 - creates the ObjectBucketClaim Operator (a Deployment API object)
 - creates a StorageClass with `parameters[serviceName]=objectstore-service` and `parameters[serviceNamespace]=some-namespace`
+  - `provisioner` field is left blank.  Since only the ObjectBucketClaim operator is watching ObjectBucketClaim instances, defining the operator is unnecessary. 
```

Comment:
...defining the operator_/provisioner_ is unnecessary and thus should be left blank.